PlotColumnDataLabelsOptions.y should be a number

#### Expected behaviour
`y` in `dataLabels` (PlotColumnDataLabelsOptions) https://api.highcharts.com/highcharts/plotOptions.column.dataLabels.y should be declared as a number in typescript definitions.

#### Actual behaviour
Compile-time error for any numeric `y`:
Type 'number' is not assignable to type 'string'.

#### Live demo with steps to reproduce
Can be reproduced in WebStorm or VSCode
```
> npm i typescript@3 highcharts@8
> echo "import * as Higcharts from 'highcharts';Higcharts.chart({plotOptions: {column: {dataLabels: {y: 0}}}})" > app.ts
> npx tsc app.ts
app.ts:7:17 - error TS2322: Type 'number' is not assignable to type 'string'.

7                 y: 0
                  ~
```

#### Product version
Highcharts 8.1.1

--- a/js/parts/ColumnSeries.js
+++ b/js/parts/ColumnSeries.js
@@ -336,6 +336,12 @@ seriesType('column', 'line',
     dataLabels: {
         align: void 0,
         verticalAlign: void 0,
+        /**
+         * The y position offset of the label relative to the point in
+         * pixels.
+         *
+         * @type {number}
+         */
         y: void 0
     },
     /**

--- a/ts/parts/ColumnSeries.ts
+++ b/ts/parts/ColumnSeries.ts
@@ -473,6 +473,13 @@ seriesType<Highcharts.ColumnSeries>(
         dataLabels: {
             align: void 0,
             verticalAlign: void 0,
+
+            /**
+             * The y position offset of the label relative to the point in
+             * pixels.
+             *
+             * @type {number}
+             */
             y: void 0
         },
 

